openstf/pipeline/utils.py

```diff
@@ -1,17 +1,26 @@
 # SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
 #
 # SPDX-License-Identifier: MPL-2.0
-from datetime import datetime, timedelta
+from datetime import datetime
+import pandas as pd
 from typing import Tuple
+import scipy.ndimage.measurements as mnts
 
 
 def generate_forecast_datetime_range(
-    resolution_minutes: int, horizon_minutes: int
+    forecast_data: pd.DataFrame,
 ) -> Tuple[datetime, datetime]:
-    # get current date and time UTC
-    datetime_utc = datetime.utcnow()
-    # Datetime range for time interval to be predicted
-    forecast_start = datetime_utc - timedelta(minutes=resolution_minutes)
-    forecast_end = datetime_utc + timedelta(minutes=horizon_minutes)
+    """Generate forecast range based on last cluster of null values in forecast data."""
```

Comment:
A slightly more detailed docstring would be very helpful for someone looking at this code for the first time:)

Reply:
Added more detailed dosctring with an example :)